webapp/packages/plugin-sql-editor/src/SqlResultTabs/OutputLogs/ACTION_CLEAR_OUTPUT_LOGS.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '@cloudbeaver/core-view';
+
+export const ACTION_CLEAR_OUTPUT_LOGS = createAction('action-clear_output_logs', {
```

Comment:
better to rename it like this:
`ACTION_{SCOPE}_NAME` -> `ACTION_OUTPUT_LOG_CLEAR`